src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -620,6 +620,7 @@ void GeneralPage::initLayout(SettingsLayout &layout)
     layout.addCheckbox("Vanity (prime, bits, subgifter)",
                        getSettings()->showBadgesVanity);
     layout.addCheckbox("Chatterino", getSettings()->showBadgesChatterino);
+    layout.addCheckbox("FrankerFaceZ", getSettings()->showBadgesFfz);
```

Comment:
```suggestion
    layout.addCheckbox("FFZ Supporter", getSettings()->showBadgesFfz);
```

Reply:
FFZ badges include not only supporters, but bots and FFZ Developers.

Reply:
Ahh that makes sense, I didn't realise. So should we then list which badges that includes in brackets? (like the authority, etc badges)

Reply:
Resolved in dca2c37